### DISCUSSION TITLE
Add a "Highlight untradeable items" toggle for Ground Items

### DISCUSSION THREAD

**Discussion by traddmccorkle** [runelite/runelite#16179] [Ideas]
**Suggestion:** Add a toggle to Ground Items, to allow all untradeable items to be highlighted

**Why:** Usually untradeable drops are desired, and adding a toggle is easier than manually entering a list of untradeables to highlight

**Comment by Enriath** [comment]
There actually aren't that many untradeables that most players would really want to have highlighted, but there are a lot of random untradeable items that people would not want highlighted. For users wanting to farm certain ones (e.g. Giant/Mossy keys), it's trivial to add to the manual highlight list, either through typing it out, or by using Alt and clicking on it while it's on the ground.

**Reply by traddmccorkle** [reply]
Fair enough
